experiments/MultiExperimentRegression/src/PetriRegression.jl

```diff
@@ -0,0 +1,52 @@
+module PetriRegression
+
+using Turing, Distributions, DifferentialEquations
+
+using MCMCChains, Plots, StatsPlots
+
+using AlgebraicPetri
+
+using Catlab.CategoricalAlgebra.CSets
+
+struct ReactionData
+  dt :: Float64
+  # Measurements of concentrations of each species at each time
+  measurements :: Array{Float64,2}
+  # Uniform priors over the rates
+  priors :: Vector{Tuple{Float64,Float64}}
+end
+
+Turing.setadbackend(:forwarddiff)
+
+@model function fit_rates(measurements, prob1)
+  σ ~ InverseGamma(2,3)
+
+  p ~ product_distribution([Uniform(0,10) for i in 1:3])
+
+  prob = remake(prob1,p=p)
+  predicted = solve(prob,Tsit5(),saveat=0.1)
+
+  for i = 1:length(predicted)
+    measurements[:,i] ~ MvNormal(predicted[i], σ)
+  end
+end
+  
+lv = PetriNet(2,(1,(1,1)),((1,2),(2,2)),(2,()))
```

Comment:
This code might be more readable if you use the supported `=>` notation for defining the transitions

```julia
lv = PetriNet(2, 1=>(1,1), (1,2)=>(2,2), 2=>())
```